### DISCUSSION TITLE
How to run this application system through the warehouse source code?

### DISCUSSION THREAD

**Discussion by 8debug** [Budibase/budibase#6027] [Help] [answered]
How to run this application system through the warehouse source code? There are no instructions in the documentation about running the system from repository source code

**Comment by mike12345567** [comment, marked as answer]
HI @8debug - have a look at our [contributing documentation](https://github.com/Budibase/budibase/blob/develop/.github/CONTRIBUTING.md).

**Reply by 8debug** [reply]
Hi, would you consider adding this document to the https://docs.budibase.com/docs/hosting-methods URL, so that people who have the same confusion as me don't have to ask in the issue

**Reply by mike12345567** [reply]
Hi @8debug - we've added a `Running Budibase` section to the docs which links to the contributing docs.
